scripts/narrative_backend_tests.sh

```diff
@@ -1,9 +1,26 @@
-#!/bin/sh
+#!/bin/bash
 export NARRATIVE_DIR=$(pwd)
+
+docker_container=${DOCKER_CONTAINER:-false}
+
+# if the test is being run in a docker container, ensure
+# that the conda base environment is activated
+if [ "$docker_container" == "true" ]; then
+    source activate base
+fi
```

Comment:
The backend tests can now either be run locally or within a docker container; the DOCKER_CONTAINER env var (set in the Dockerfile) will ensure that the correct environment  is activated and will copy the coverage file to an appropriate location for it to be shared outside the docker container.